tests/ui/consts/const-mut-refs/issue-76510.stderr

```diff
@@ -4,17 +4,6 @@ error[E0764]: mutable references are not allowed in the final value of constants
 LL | const S: &'static mut str = &mut " hello ";
    |                             ^^^^^^^^^^^^^^
```

Comment:
I wish we had a more complete error here :)

Reply:
What would a more complete error look like?